third_party/subscriptions-project/swg.js

```diff
@@ -2270,10 +2327,10 @@ const TITLE_LANG_MAP = {
 class ButtonApi {
 
   /**
-   * @param {*} doc
+   * @param {!../model/doc.Doc} doc
```

Comment:
"Cannot find module" is an error in newer CC. I replaced these with `*` with a manual regex search/replace:

https://github.com/ampproject/amphtml/blob/493ea5fd3915b9e5efd9b070bf71d0dc3d48e424/third_party/subscriptions-project/README.amp#L10

#18586 for details.